docs/tutorials/elasticdl_local.md

```diff
@@ -1,82 +1,99 @@
+Minikube.
 
-## Environment preparation
+## Prerequisites
 
-1. Install Minikube >= v1.11.0. Please refer to the official
-[installation guide](https://kubernetes.io/docs/tasks/tools/install-minikube/).
-In this tutorial, we use [hyperkit](https://github.com/moby/hyperkit) as the
-hypervisor of Minikube.
-1. Install [Docker CE >= 18.x](https://docs.docker.com/docker-for-mac/install/)
-for building the Docker images of the distributed ElasticDL jobs.
-1. Install Python >= 3.6.
+1. Install Minikube, preferably >= v1.11.0, following the [installation
+   guide](https://kubernetes.io/docs/tasks/tools/install-minikube).  Minikube
+   runs a single-node Kubernetes cluster in a virtual machine on your personal
+   computer.
 
-## Write model file
+Minikube works with a variety of virtual machine hypervisors.  In this tutorial,
+we use [hyperkit](https://github.com/moby/hyperkit) that comes with macOS. If
+you want to use other hypervisors like VirtualBox, please install them
+accordingly.
 
-We use TensorFlow Keras API to build our models. Please refer to this
-[tutorials](model_building.md) on model building for details.  In this tutorial,
-we use a model predefined in model zoo directory.
+1. Install Docker CE, preferably >= 18.x, following the
+   [guide](https://docs.docker.com/docker-for-mac/install/) for building Docker
+   images containing user-defined models and the ElasticDL framework.
 
-## Summit Job to Minikube
+1. Install Python, preferably >= 3.6, because the ElasticDL command-line tool is
+   in Python.
 
-### Install ElasticDL Client
+## Models
+
+Among all machine learning toolkits that ElasticDL can work with, TensorFlow is
+the most tested and used.  In this tutorial, we use a model from the [model
+zoo](https://github.com/sql-machine-learning/elasticdl/tree/develop/model_zoo)
+directory.  This model is defined using TensorFlow Keras API.  To write your
+models, please refer to this [tutorial](./model_contribution.md).
+
+## Datasets
+
+We use the MNIST dataset in this tutorial.  The dynamic data partitioning
+mechanism of ElasticDL requires that the training data files are in the
+[RecordIO format](https://pypi.org/project/pyrecordio). To download the MNIST
+dataset and convert it into RecordIO files, please run the following command.
 
 ```bash
-pip install elasticdl_client
+docker run --rm -it \
+  -v $HOME/.keras:/root/.keras \
+  -v $PWD:/work \
+  elasticdl/elasticdl:dev /scripts/gen_dataset.sh data
 ```
 
-Clone elasticdl repo for model zoo and some scripts.
+After the running of this command, we will see dataset files in the current
+directory.
+
+## The Kubernetes Cluster
+
+The following command to start a Kubernetes cluster locally using Minikube.
 
 ```bash
-git clone https://github.com/sql-machine-learning/elasticdl.git
+minikube start --vm-driver=hyperkit \
+  --cpus 2 --memory 6144 --disk-size=50gb \
+  --mount=true --mount-string="./data:/data"
+kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
```

Comment:
We could package `elasticdl-rbac.yaml` into `elasticdl-client` wheel package, so that users need not clone elasticdl source code to get this file.